src/main/features/claim/views/amount.njk

```diff
@@ -11,6 +11,8 @@
       <form novalidate method="post">
         {{ csrfProtection(csrf) }}
 
+        <input type="hidden" id="focus-target" value="{{ form.postbackFocusTarget | default('') }}" />
```

Comment:
I do not like the idea of having default value with empty string here. What about conditional `{% if form.action %}`? In that case we would just do `form.postbackFocusTarget`